platforms/documentation/docs/src/docs/userguide/reference/ci-systems/gitlab.adoc

```diff
@@ -0,0 +1,211 @@
+
+[listing]
+----
+build-gradle-project:
+  image: eclipse-temurin:latest
+
+  script:
+    - ./gradlew build
+
+  artifacts:
+    reports:
+      junit: '**/build/test-results/**/TEST-*.xml'
+----
+
+This file defines a single job that executes the `./gradlew build` command.
+
+* The `image` field defines which Docker image the job will run in. Thanks to the Gradle Wrapper, only a JDK is necessary.
+* The `artifacts` field defines files generated by the job that GitLab CI should save. In particular, saving the JUnit report generated by the Gradle test tasks allows displaying test results in the Merge Request or Pipeline pages directly.
+
+Commit the changes and push to the repository:
+
+[listing.terminal.sample-command]
+----
+$ git add .
+$ git commit -m "Add GitLab CI pipeline"
+$ git push
+----
+
+== View the GitLab CI results
+
+Once this pipeline file is pushed, you should immediately see the pipeline execution in the GitLab CI page for your repository (eg https://gitlab.com/username/repository/-/pipelines).
+Any subsequent push to the repository will trigger the pipeline to run.
+
+== Enable caching
+
+By default, no files are persisted between CI runs, so each job must download all dependencies again. We can configure GitLab to cache some files between jobs. This is especially efficient when running your own GitLab CI runner, as the cache is stored locally on the runner.
+
+CAUTION: The publicly available shared runners store their cache in a remote location. As the cache grows, downloading the cache archive, extracting it, and uploading it again at the end of the job may quickly take more time than not using any cache at all. For this reason, we recommend only caching the Gradle Wrapper. Prefer configuring a link:https://docs.gradle.org/current/userguide/build_cache.html[Gradle Remote Build Cache] for caching everything else.
+
+To reuse some files between jobs, add the following configuration to your existing jobs:
+[listing]
+----
+build-gradle-project:
+  # …existing configuration…
+
+  variables:
+    GRADLE_USER_HOME: $CI_PROJECT_DIR/.gradle
+
+  cache:
+    paths:
+      - .gradle/wrapper
+    key:
+      files:
+        - gradle/wrapper/gradle-wrapper.properties
+----
+
+Commit the changes and push to the repository:
+
+[listing.terminal.sample-command]
+----
+$ git add .
+$ git commit -m "Cache the Gradle Wrapper between CI jobs"
+$ git push
+----
+
+== Further reading
+
+Learn more about building Gradle projects with GitLab CI:
+
+* https://docs.gitlab.com/ee/ci/[GitLab CI documentation]
+* https://docs.gitlab.com/ee/ci/yaml/[.gitlab-ci.yml syntax reference]
+* https://docs.gitlab.com/ee/ci/variables/predefined_variables.html[Predefined variables]
+* https://docs.gitlab.com/ee/ci/yaml/yaml_optimization.html[Avoiding duplication between multiple jobs]
+
+== Summary
+
+Executing Gradle builds on CI can be set up and configured with just a handful of steps.
+The benefit of receiving fast feedback clearly speaks for itself.
```

Comment:
```suggestion
```

Reply:
This sentence is present in all other pages.